### PR TITLE
Bug 1344393 added info on creating scc with requireddropcapabilities field

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1344393

Will ask for ack over email.

### CODE REVIEW

**Comment by bfallonf** [comment]
Ack given.

@tpoitras @ahardin-rh @adellape Peer review please?

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/manage_scc.html#creating-new-security-context-constraints[Cluster Administration -> Managing Security Context Constraints]
|Added information about required drop capabilities to the xref:../admin_guide/manage_scc.html#creating-new-security-context-constraints[Creating New Security Context Constraints] section.
%

**Comment by bfallonf** [comment]
bump @ahardin-rh @adellape @tpoitras

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -128,14 +131,39 @@ groups:
 - my-admin-group
 ----
 ====
++
+Optionally, you can add drop capabilities to an SCC by setting the
+`*requiredDropCapabilities:*` field with the desired values. Any specified
+capabilities will be dropped from the container. For example, to create an SCC
+with the `*KILL*`, `*MKNOD*`, and `*SYS_CHROOT*` required drop capabilities add
```
Comment (ahardin-rh) [inline admin_guide/manage_scc.adoc]: comma after capabilities, maybe?

**Comment by ahardin-rh** [comment]
@bfallonf just one comment from me :star:

**Comment by bfallonf** [comment]
@ahardin-rh Big thanks! I'll merge.

**Pull request merged**